File: movie/movie-adapter/src/main/java/com/bmsnc/adapter/MovieApplication.java
package com.bmsnc.adapter;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.ComponentScans;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@ComponentScan("com.bmsnc")
@EnableJpaRepositories("com.bmsnc")
public class MovieApplication {
    public static void main(String[] args) {
        SpringApplication.run(MovieApplication.class, args);
    }
}

File: movies/src/main/kotlin/com/redis/MovieApplication.kt
package com.redis

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class MoviesApplication
fun main(args: Array<String>) {
    runApplication<MoviesApplication>(*args)
}